frontend/e2e/application-noncommercial-group.e2e-spec.ts

```diff
@@ -44,21 +44,9 @@ describe('Apply for a noncommercial group use permit', () => {
     expect<any>(element(by.css('app-root h1')).getText()).toEqual('Apply for a noncommercial group use permit.');
   });
   fieldValidation.validateSimpleTextField('name', 'name-error', 'Event name is required.');
-  fieldValidation.validateSimpleTextField(
-    'primary-permit-holder-first-name',
-    'primary-permit-holder-first-name-error',
-    'Primary permit holder first name is required.'
-  );
-  fieldValidation.validateSimpleTextField(
-    'primary-permit-holder-last-name',
-    'primary-permit-holder-last-name-error',
-    'Primary permit holder last name is required.'
-  );
-  fieldValidation.validateSimpleTextField(
-    'primary-permit-holder-address',
-    'primary-permit-holder-address-error',
-    'Street address 1 is required.'
-  );
+  fieldValidation.validateSimpleTextField('primary-permit-holder-first-name', 'primary-permit-holder-first-name-error', 'Primary permit holder first name is required.');
```

Comment:
We've shortened our prettier line lengths to 120, so you should start seeing these broken out into a couple lines.